geode-core/src/main/java/org/apache/geode/distributed/internal/InternalDistributedSystem.java

```diff
@@ -817,6 +807,34 @@ private void initialize(SecurityManager securityManager, PostProcessor postProce
     reconnectAttemptCounter.set(0);
   }
 
+  void lockMemory(long avail, long size) {
+    if (avail < size) {
+      if (avoidMemoryLockWhenOvercommitted) {
+        logger.warn(
+            "System memory appears to be over committed by {} bytes. Avoid memory lock.",
```

Comment:
Wouldn't this experience similar issues to locking memory, but just without locking memory?  Meaning, should the two messages be the same?

Reply:
No it would not experience the issue if locking memory. Without locking memory, memory can be paged in and out -- yes, it may have performance impact but should not experience instability issue, etc.